src/array/__tests__/arrayEqual-test.js

```diff
@@ -0,0 +1,18 @@
+import arrayEqual from '../arrayEqual';
+import { expect } from 'chai';
+
+describe('arrayEqual', () => {
+    it('Compare two given arrays unequal arrays', () => {
+        const arr1 = [10,20,30, 500, -1];
```

Comment:
Try to make arrays as simple as possible [1, 2, 3] for example.
You can use -1 in one of the cases if you want to demonstrate it works with negatives.